client/src/components/Event.jsx

```diff
@@ -86,8 +86,8 @@ class Event extends React.Component {
 
   getEventContent () {
     axios.get(`/api/events/${this.state.eventId}`)
-    .then(({ data: { title, delivery_time, recipient} }) => {
-      this.setState({ title, delivery_time, recipient });
+    .then(({ data: { title, deliveryTime, recipient} }) => {

```

Comment:
this needs to change. delivery_time is what comes from the response, but deliveryTime is the state. So, this should be {title, delivery_time, recipient } and then the line below needs to be {title, deliveryTime: delivery_time, recipient} (I think).